.gitignore

```diff
@@ -1,2 +1,3 @@
 /vendor
 composer.lock
+.idea/*
```

Comment:
I suggest (as [I was suggested before](https://github.com/skrz/autowiring-bundle/pull/2/files#r35136818)) to use `~/.gitignore_global` to ignore IDE's projects files. This is not necessary.